app/components/login/login.html

```diff
@@ -0,0 +1,66 @@
+<div id="alertMessage" ng-show="loginCtrl.errorMsg" class="alert alert-danger alert-dismissible fade in"
+     role="alert"><a class="close" data-dismiss="alert" aria-label="close">&times;</a>
+    <p>{{loginCtrl.errorMsg}}</p>
+</div>
+
+<div class="row">
+    <div class="col-md-10 col-md-offset-1">
+        <div class="panel panel-login">
+            <div class="panel-heading">
+                <div class="row">
+                    <div class="col-xs-6">
+                        <a class="active">Log In</a>
+                    </div>
+                    <div class="col-xs-6">
+                        <a ui-sref="register">Register</a>
+                    </div>
+                </div>
+                <hr>
+            </div>
+            <div class="panel-body">
+                <div class="row">
+                    <div class="col-lg-12">
+
+                        <!--Login Form-->
+                        <form id="login-form" role="form" ng-submit="loginCtrl.validateUser(loginCtrl.user)">
+                            <div class="col-sm-6 col-sm-offset-3">
+                                <div class="form-group">
+                                    <input type="text" name="username" tabindex="1" class="form-control"
+                                           placeholder="User Email" ng-model="loginCtrl.user.email">
+                                </div>
+                                <div class="form-group">
+                                    <input type="{{loginCtrl.inputType}}" name="password" tabindex="2"
```

Comment:
Use static text `password` for `type` attribute, remove `{{loginCtrl.inputType}}`